tests/test_package.py

```diff
@@ -82,7 +82,7 @@ def test_wheel_package():
 
 
 @pytest.mark.parametrize("package_name", [
-  "transmute_core-0.4.5-py2.py3-none-any-gobbledy.whl"
+  "transmute_core-0.4.5-py2.py3-none.whl"
```

Comment:
Had to change this test because we're now using `rsplit`, so this invalid name would now sneak by as valid. I think this types of screw ups are incredibly unlikely to occur as the wheel names aren't manually created.